### DISCUSSION TITLE
Silent fit mode for AutoModel

### DISCUSSION THREAD

**Discussion by RinnetenseiQ** [keras-team/autokeras#1773] [Q&A] [answered]
How do I remove validation progress bar while training to get absolute silent fitting mode?
![verbose](https://user-images.githubusercontent.com/49791012/188633740-fa3a0ae5-c2b6-4e1f-9cbb-f11ac12fbb72.PNG)

**Comment by haifeng-jin** [comment, marked as answer]
This is a bug. The `verbose=0` should be passed down to both `model.fit()` and `model.predict()` for the Keras model under the hood. Seems now it only passed down to `model.fit()`. Would you like to help us fix the bug?

**Reply by RinnetenseiQ** [reply]
I want to use tqdm progressbar outside and internals pb will interfere. But I don't have much experience to edit another's sourses of code. And I also observe, that counts of pbs grows with the number of epochs

**Reply by RinnetenseiQ** [reply]
![image](https://user-images.githubusercontent.com/49791012/190165734-8d74b139-851f-4fe0-91d8-0288f6642eb4.png)
As I think, verbose flag should be passed in `final_fit()` method

**Reply by haifeng-jin** [reply]
The PR is merged. It should be fixed in the next release. or you can use master branch now.
